stdpopsim/catalog/homo_sapiens.py

```diff
@@ -263,11 +266,12 @@ def _ooa_2():
         stdpopsim.Population(name="EUR", description="European Americans")
     ]
     citations = [
-        _tennessen_et_al,
+        _tennessen_et_al.because(stdpopsim.CiteReason.DEM_MODEL),
```

Comment:
seems like using `.because` here is a break in the style. we should set the reason when `_tennessen_et_al` is defined IMO

Reply:
Yep, thanks. Updated.